internal/agent/status/status_test.go

```diff
@@ -230,3 +276,81 @@ func TestNewReporter_ReportEntity(t *testing.T) {
 		})
 	}
 }
+
+//nolint:paralleltest
+func TestNewReporter_ReportHealth(t *testing.T) {
+	serverOk := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(http.StatusOK)
+	}))
+	defer serverOk.Close()
+
+	serverTimeout := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		time.Sleep(10 * time.Second)
```

Comment:
Nitpick: This ☝️ makes the test take 10 seconds. It could be changed to
```go
time.Sleep(100 * time.Millisecond)
```

Reply:
Fixed in last commit.